docs/dictionary/command/sort-container.lcdoc

```diff
@@ -49,50 +52,52 @@ sortType (enum):
 If you don't specify a <sortType>, the <sortType> is `text`.
 
 -   international: sorts by collation according to the system locale
--   numeric: sorts by number. (Use this form if the sortKey consists of
-    numbers) 
--   datetime: treats the sortKey as a date and/or time
+-   numeric: sorts by number. (Use this form if the sortKey consists
+    of numbers) 
```

Comment:
Need to check how this is rendered.  I'm seeing the continuation line show up in the wrong place (at the end of the text in line 52).  I tried to move the "of" as above and saw the same thing.

Reply:
Yep, you're right. Somehow I missed that obvious problem as I was fixing a bunch of other small things. I'm going to cancel this PR and do some more playing around with it. Panos asked me to rebase it on develop-8.2 anyway.

Reply:
Thanks guys. Yes, `develop-8.2` is now the recommended branch for docs changes (https://github.com/livecode/livecode/pull/5911)